### PR TITLE
feat: update the donation banner

### PR DESCRIPTION
**Pull request by Valimp**
### What

Update the donation banner with 2024 design

### Screenshot
ENG
FR

### CODE REVIEW

**Comment by Valimp** [comment]
## Screenshots

### Desktop

Top banner : 
Capture d’écran 2024-10-23 à 16 06 21

Footer banner :
Capture d’écran 2024-10-23 à 16 06 39


### Mobile

Top banner : 
Capture d’écran 2024-10-23 à 16 07 25

Footer banner :
Capture d’écran 2024-10-23 à 16 07 54

**Review by teolemon — COMMENTED** [review]
- I'm not a very big fan of hardcoded images that don't scale translation wise
- and changing the texts this year (which deletes all preexisting translations)

**Comment by Valimp** [comment]
Capture d’écran 2024-10-28 à 10 55 59

File: po/common/en.po [hunk 1]
```diff
@@ -5835,6 +5835,46 @@ msgctxt "donation_text_2023_tertiary"
...
+msgid "support the advancement of public health research."
+msgstr "support the advancement of public health research."
+
+msgctxt "donation_financial_2024_text"
+msgid "Each donation counts ! We appreciate your support in bringing further food transparency in the world."
+msgstr "Each donation counts ! We appreciate your support in bringing further food transparency in the world."
+
+msgctxt "donation_button_2024_text"
+msgid "I support"
+msgstr "I SUPPORT"
```
Comment (stephanegigandet) [inline po/common/en.po:5876]: in English, msgstr should be exactly the same as msgid

File: po/common/en.po [hunk 2]
```diff
@@ -5835,6 +5835,46 @@ msgctxt "donation_text_2023_tertiary"
 msgid "The food revolution starts with you!"
 msgstr "The food revolution starts with you!"
 
+msgctxt "donation_hook_2024"
```
Comment (stephanegigandet) [inline po/common/en.po:5838]: all strings added to en.po also need to be added to common.pot

**Comment by Valimp** [comment]
@stephanegigandet All checks have passed, and I fixed what you ask me

**Comment by stephanegigandet** [comment]
@Valimp I made a few changes:

added a "row" class so that the banner is not too large on very wide screens
aligned right column to the left, removed extra line in sub list element
changed a few sentences in the .po (mostly punctation)
made the 66% deduction only shown when the country is France

image

**Pull request merged**